Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -65,11 +65,15 @@ class BookmarksPanel: SiteTableViewController, HomePanel {
     override func viewDidLoad() {
         super.viewDidLoad()
         self.tableView.accessibilityIdentifier = "Bookmarks List"
+        self.setupBookmarkSource()
```

Comment:
It was added in this PR here https://github.com/mozilla-mobile/firefox-ios/pull/2193/commits/6144265d02fe20684afd1af23bca0892fd349004 @sleroux any thoughts why?